## Amyloid bioinformatics

### Questions

How are amyloid diseases the same, and how are they different?

More specifically:

1) What molecular pathways are common to many or all 
          amyloid diseases?
2) What molecular pathways are unique to certain amyloid 
          diseases (such as prion disease or Alzheimer's disease?)

### Rationale:

A bioinformatics approach allowing comparisons of gene expression, gene annotation, and protein-protein and protein-DNA interaction data may answer these questions and provide leads for biomarker discovery

### Aims and Methods 

1) Comparative microarray analysis: 
      Create a unified amyloid bioinformatics database using gene transcription data
      
      Outcome: two target lists of genes of interest â€“ by their expression across many 
         amyloid diseases, or by expression unique to individual amyloid diseases
         
      ***Completed proof-of-concept through this stage***

2) Ontology annotation and data integration: 
      Create a semi-automated text mining and gene annotation tool using known 
         ontologies, as well as with associated gene-disease, protein-DNA and protein-
         protein interaction data
      
      Outcome: function annotation and clustering of genes in the target lists above

3) Data visualization and network inference: 
      Create a data visualization and network inference tool, using as input the unified 
          amyloid database and the ontology annotation tool described above
      
      Outcome: concise data visualization and network inference to prioritize target 
          lists of genes and gene products for further analysis as biomarkers
